controller/node_controller.go

```diff
@@ -1018,3 +1001,215 @@ func BackingImageDiskFileCleanup(node *longhorn.Node, bi *longhorn.BackingImage,
 		delete(bi.Spec.Disks, diskUUID)
 	}
 }
+
+func (nc *NodeController) checkMonitor(node *longhorn.Node) (monitor.Monitor, error) {
+	if node == nil {
+		return nil, nil
+	}
+
+	if nc.monitor != nil {
+		return nc.monitor, nil
+	}
+
+	monitor, err := monitor.NewNodeMonitor(nc.logger, nc.ds, node, nc.enqueueNodeForMonitor)
```

Comment:
1. Will the node controller take over the unknown node CR?
2. Will it create monitors for those unknown node CRs?

Reply:
What's the unknown node CRs?
Do you mean  `nc.controllerID != node.Name`?
If yes, won't create monitors for the unknown node CRs.